### PR TITLE
chore: update deps and fix audit

### PR DESCRIPTION
**Pull request by DiegoRBaquero**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,13 +37,13 @@
     "uuid": "3.x.x"
   },
   "devDependencies": {
-    "@pager/semantic-release-config": "1.x",
+    "@pager/semantic-release-config": "1.3.0",
```
Comment (duvanmo) [inline package.json:40]: keep the 1.x

File: package.json [hunk 2]
```diff
@@ -37,13 +37,13 @@
...
     "chai": "4.x.x",
     "eslint": "6.x.x",
     "eslint-config-hapi": "12.x.x",
     "eslint-plugin-hapi": "4.x.x",
-    "mocha": "7.x",
-    "semantic-release": "17.x"
+    "mocha": "7.1.1",
```
Comment (duvanmo) [inline package.json:45]: same here 7.x

File: package.json (hunk 1 above)
Comment (jugaltheshah) [inline package.json:40]:
```suggestion
    "@pager/semantic-release-config": "1.x.x",
```
Same for `mocha` and `semantic-release`.

File: package.json [hunk 3]
```diff
@@ -37,13 +37,13 @@
...
+    "semantic-release": "17.0.6"
```
Comment (duvanmo) [inline package.json:46]: same here 17.x

File: package.json (hunk 1 above)
Comment (rshah1207) [inline package.json:40]: Why are we moving away from semver for these ?

**Pull request merged**